src/vuestic-theme/vuestic-sass/grid/_grid-mixins.scss

```diff
@@ -24,6 +24,7 @@
 }
 
 @mixin va-container--fluid {
+  box-sizing: border-box;
```

Comment:
Any rationale behind this solution?

Grids are a bit tightly knit. So changing them might break things in unpredictable way.

Reply:
Things are all checked for unpredictable breaks (for the most part at least), and that's quite logical for containers to be border-box sized (imho). By the way, it well may be implemented so as well in bootstrap and that's definetely better than calculating things manually. No cons afaics.